src/containers/AssetList.js

```diff
@@ -97,7 +97,7 @@ const AssetList = ({ match }) => (
         >
           {assetData.map( (asset, index) => (
             <AssetListItem
-              index={asset.index}
+              key={index}
```

Comment:
When we have a real API, this should be asset.id or similar. For the moment, this is fine.